src/utils/tranche.utils.ts

```diff
@@ -491,3 +493,52 @@ export const investorCurrentReturns = (tranche: Tranche, userTrancheBalance: Tok
 		Big.roundHalfEven
 	);
 };
+
+interface TrancheCSVRecord {
+	tranche: string;
+	size: string;
+	return: string;
+}
+
+/**
+ * Parses a CSV string into a DefaultTranche. The first record of the csv should be the header.
+ * @param input  A CSV string to be parsed, which should have the follwing format: tranche,size,return
+ * @returns the parsed CSV string as a DefaultTranche
+ */
+export const parseTrancheCSV = (input: string): DefaultTranche => {
+	const records: TrancheCSVRecord[] = parse(input, {
+		skip_empty_lines: true,
+		trim: true,
+		columns: (header) => header.map((column: string) => column.toLowerCase()),
```

Comment:
this will cause the first row to be used as column names even if it is just a record. I would just declare the expected column names as an array and let it fail if the user doesn't follow it.